build.gradle

```diff
@@ -446,6 +448,7 @@ project(':cruise-control-metrics-reporter') {
     implementation "com.yammer.metrics:metrics-core:2.2.0"
     implementation "org.apache.logging.log4j:log4j-slf4j-impl:2.17.2"
     implementation "org.apache.kafka:kafka_$scalaBinaryVersion:$kafkaVersion"
+    implementation "org.apache.kafka:kafka-server:$kafkaVersion"
```

Comment:
with this, `com.linkedin.cruisecontrol:cruise-control-metrics-reporter` artifact dependencies would be much heavier than I wanted, I'm checking if I can reduce it

Reply:
nevermind, it's pulled in by the kafka_<scala version> on runtimeClasspath, but required on implementation as well, can't spare it